src/main/scala/com/cognite/sdk/scala/v1/fdm/views/ViewPropertyCreateDefinition.scala

```diff
@@ -1,13 +1,16 @@
 package com.cognite.sdk.scala.v1.fdm.views
 
 import cats.implicits.toFunctorOps
-import com.cognite.sdk.scala.v1.fdm.common.DirectRelationReference
+import com.cognite.sdk.scala.v1.fdm.common.properties.PropertyDefinition.{
+  ConnectionDefinition,
+  connectionDefinitionDecoder
+}
 import com.cognite.sdk.scala.v1.fdm.containers.ContainerReference
-import io.circe.{Decoder, Encoder}
 import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
 import io.circe.syntax.EncoderOps
+import io.circe.{Decoder, Encoder}
 
-sealed trait ViewPropertyCreateDefinition
+trait ViewPropertyCreateDefinition
```

Comment:
why do we need to unseal it? If all implementors are in sdk it should be possible to keep it sealed I think

Reply:
Sealed trait can only be implemented in the same file. In this case, I want the ConnectionConfiguration to also implement the ViewPropertyCreateDefinition because when creating a view you can have a connection, so it's either this, copy all in the same file, or have a code duplication in this file